google/apistore.go

```diff
@@ -85,3 +87,52 @@ func (c *APIStore) SetBucketAgeLifecycle(name string, days int64) error {
 	_, err := c.service.Buckets.Patch(name, bucket).Do()
 	return err
 }
+
+// GrantObjectViewer updates the IAM policy on the bucket to grant member the roles/storage.objectViewer role
+// The existing policy attributes on the bucket are preserved
+func (c *APIStore) GrantObjectViewer(bucket, member string) error {
+	return c.grantRole(bucket, member, "roles/storage.objectViewer")
+}
+
+// GrantObjectCreator updates the IAM policy on the bucket to grant member the roles/storage.objectCreator role
+// The existing policy attributes on the bucket are preserved
+func (c *APIStore) GrantObjectCreator(bucket, member string) error {
+	return c.grantRole(bucket, member, "roles/storage.objectCreator")
+}
+
+// GrantObjectAdmin updates the IAM policy on the bucket to grant member the roles/storage.objectAdmin role
+// The existing policy attributes on the bucket are preserved
+func (c *APIStore) GrantObjectAdmin(bucket, member string) error {
+	return c.grantRole(bucket, member, "roles/storage.objectAdmin")
+}
+
+func (c *APIStore) grantRole(bucket, member, role string) error {
+	existingPolicy, err := c.service.Buckets.GetIamPolicy(bucket).Do()
+	if err != nil {
+		return err
+	}
+
+	var added bool
+	for _, b := range existingPolicy.Bindings {
+		if b.Role == role {
```

Comment:
[nit] not a big deal but do you want to do `strings.IgnoreCase` just to be safe if there were any character case change in the future.

Reply:
yeah, for now this is just for google-define roles, which are case sensitive, so think it's ok to leave for now